Bindings/Java/OpenSimJNI/OpenSimContext.h

```diff
@@ -221,7 +221,13 @@ OpenSim_DECLARE_CONCRETE_OBJECT(OpenSimContext, Object);
 
     void cacheModelAndState();
     void restoreStateFromCachedModel()  SWIG_DECLARE_EXCEPTION;
-    void setSocketConnecteeName(AbstractSocket& socket, const std::string& newValue )   SWIG_DECLARE_EXCEPTION;
+    void setSocketConnecteePath(AbstractSocket& socket,
+            const std::string& newValue)   SWIG_DECLARE_EXCEPTION;
+    // Deprecated.
+    void setSocketConnecteeName(AbstractSocket& socket,
```

Comment:
No need to keep this , will update the GUI code base. it's used in only a couple places.